ui/pages/settings/flask/view-snap/view-snap.js

```diff
@@ -71,6 +72,10 @@ function ViewSnap() {
   if (!snap) {
     return null;
   }
+
+  const versionHistory = snap.versionHistory ?? [];
+  const firstInstall = versionHistory[0];
```

Comment:
I've been trying to get the codebase in the habit of:

```
const [firstInstall] = versionHistory;
```

Reply:
Done 👍